src/main.cpp

```diff
@@ -102,6 +122,30 @@ void setup()
         }
         delay(100);
     }
+    //set firingMode
+    if (pins.select1) {
```

Comment:
Why do we only update the select pins if their value is high? Should we not unconditionally update them so that their state is correct?

It might be okay in the setup loop, but in the main body loop, won't they never be deselected because their state is never updated?

Reply:
I based this off the existing code for rev and fire switches and have had no issues. The 3 position switch can turn on one of the pins or neither which is why I treat it as 2 switches just like rev and fire.

Reply:
My bad; that's checking if the pin exists before updating it; the code's correct